assets/js/components/DetailsPermaLinks.js

```diff
@@ -66,3 +68,13 @@ export default function DetailsPermaLinks( { title, path } ) {
 		</Fragment>
 	);
 }
+
+DetailsPermaLinks.propTypes = {
+	title: PropTypes.string,
+	path: PropTypes.string,
+	serviceURL: PropTypes.string,
+};
+
+DetailsPermaLinks.defaultProps = {
+	serviceURL: null,
+};
```

Comment:
Unless there is a reason why we need a specific value to handle this case, I think this can be removed. We generally use `null` as our intentional value for none. In this case the default `undefined` should fine if it isn't passed.

Reply:
👍